build.gradle

```diff
@@ -95,15 +105,12 @@ dependencies {
     testCompile 'eu.codearte.catch-exception:catch-exception:2.0.0-ALPHA-1'
     testCompile 'nl.jqno.equalsverifier:equalsverifier:2.3.3'
     testCompile 'org.hamcrest:java-hamcrest:2.0.0.0'
-    testCompile 'org.mockito:mockito-core:2.10.0'
-    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:2.8.47'
```

Comment:
BTW, was just curious what triggered the need to downgrade Mockito?

Reply:
Bad merge conflict resolution, will fix that ^^